seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/CoordinatorService.java

```diff
@@ -146,8 +155,11 @@ private void initCoordinatorService() {
         ownedSlotProfilesIMap = nodeEngine.getHazelcastInstance().getMap(Constant.IMAP_OWNED_SLOT_PROFILES);
 
         List<CompletableFuture<Void>> collect = runningJobInfoIMap.entrySet().stream().map(entry -> {
-            return CompletableFuture.runAsync(() -> restoreJobFromMasterActiveSwitch(entry.getKey(), entry.getValue()),
-                executorService);
+            return CompletableFuture.runAsync(() -> {
+                logger.info(String.format("begin restore job (%s) from master active switch", entry.getKey()));
```

Comment:
Use `{}` instead of String.format

https://stackoverflow.com/questions/41532840/log4j2-using-against-using-d-or-s

Reply:
> Use `{}` instead of String.format
> 
> https://stackoverflow.com/questions/41532840/log4j2-using-against-using-d-or-s

We use the Hazelcast Logger API, it not support use `{}`

<img width="611" alt="image" src="https://user-images.githubusercontent.com/32193458/195751481-c0ac80c8-27e2-41dd-aa8a-0dea350a41f6.png">